static/app/views/insights/pages/platform/nextjs/index.tsx

```diff
@@ -1,22 +1,35 @@
 import {useEffect} from 'react';
+import {useLocation, useNavigate} from 'react-router-dom';
```

Comment:
We have our own wrapper functions for those APIs.
```suggestion
import {useLocation} from 'sentry/utils/useLocation';
import {useNavigate} from 'sentry/utils/useNavigate';
```